Unity-Technologies/ScriptableRenderPipeline
# Urp mockhmd dennisd

### Checklist for PR maker
- Have you added a Label? : HDRP, Universal, ShaderGraph etc...
No.
- Have you added a label for backport (if needed)? : need-backport-2019.3  .  When the PR is backported the label will be change ton backported-2019.3
No. 
- Have you added a changelog? Each package have a changelog.
No. 
- Have you updated or added the documentation for you PR? When property name is changed, when a feature behavior is change, when adding a new features, think to update the documentation in the same PR.
N/A. 
- Have you added a graphic test for your PR (if needed)? When adding new feature or discovering a bug that isn't cover by a test, please add a graphic test
Yes, this PR is new graphics tests.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
We need more test coverage or Universal RP for XR.  I will add several new tests periodically until the Stereo automation coverage for Universal matches what we have in Katana for the BuiltIn RP.

---
### Testing status

**Manual Tests**: What did you do?
- [Yes] Opened test project + Run graphic tests locally
- [No] Built a player
- [N/A] Checked new UI names with UX convention
- [N/A] Tested UI multi-edition + Undo/Redo + Prefab overrides
- [No] C# and shader warnings (supress shader cache to see them)
- [No] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the pathes)
- Other: 
On Yamato "_ABV for SRP repository - trunk" is green for urp_mockhmd_dennisd commit b9ea471.  Here is the link. https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/urp_mockhmd_dennisd/.yamato%252Fupm-ci-abv.yml%2523all_project_ci_trunk/746124/job

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
This PR adds stereo graphics tests for reflection probes, particles, and static batching. 

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/urp_mockhmd_dennisd

Any test projects to go with this to help reviewers?
This PR adds a handful of tests to the existing UniversalGfxTestStereo project in the SRP repo. 

---
### Comments to reviewers
Notes for the reviewers you have assigned.
Sophia -- This should be easy to approve since all the changed/added files are in UniversalGfxTestStereo and ABV is green.  But, let me know if you spot anything I need to change. Thanks!
